### DISCUSSION TITLE
Has anybody used tobac for "slow" meteorological phenomena?

### DISCUSSION THREAD

**Discussion by JuliaKukulies** [tobac-project/tobac#181] [Q&A] [answered]
The input for `tobac` can be any meteorological field and the spatial and temporal scale of the target phenomenon does in theory not matter. I am curious if anybody in the community has used or plans to use  `tobac` to track "slow" phenomena (e.g. heat waves, ENSO, polynyas,...) that occur on climate-like time scales rather than on weather-like time scales? Suggestions from previous experience with slow-moving systems or any other input are welcome :)

**Comment by zxdawn** [comment]
I heard about Jetlag in the last EGU: https://lrivoire.github.io/JetLag/. But, it seems not exactly as same as you want.

**Reply by JuliaKukulies** [reply]
Thanks @zxdawn, much appreciated! Could be useful to see what the fundamental differences between `tobac` and packages that are more specifically designed for slow-moving systems are. I will have a look into this.

**Reply by zxdawn** [reply]
@JuliaKukulies You're welcome. This is another tool for tracking atmospheric river: https://github.com/ihesp/IPART

**Comment by zxdawn** [comment, marked as answer]
@JuliaKukulies I came across another package called [PyFLEXTRKR](https://github.com/FlexTRKR/PyFLEXTRKR). The example shows it can also track 500 hPa geopotential height anomaly.

**Reply by JuliaKukulies** [reply]
Thank you for pointing me to this example, @zxdawn ! I am familiar with PyFLEXTRKR, but hadn't seen this example yet
